conf/extra-modules-example.yaml

```diff
@@ -148,6 +148,18 @@ modules:
 # notifies opers of user disconnections on remote servers.
 #- ServerNoticeRemoteQuit
 
+# StatsOnlineOpers: Provides the ONLINEOPERS STATS type, which lists all opers
```

Comment:
The STATS types are implemented in lowercase, so we should have them that here as well.